### ISSUE TITLE
There is need to care various html markup style.

### ISSUE THREAD

**Issue by Beingbook** [Glavin001/atom-beautify#569]
Hello.

I coded html5(exactly AngularJS View) like this:

``` html
<element
  attr-a="{
    object: condition
  }"
  attr-b="blahblah"
  attr-etc=".."
>
```

And I beautifered it:

``` html
<element
  attr-a="{
    object: condition
  }"
  attr-b="blahblah" attr-etc="..">
```

I think above codes are needed to be maintained clearly.
But I can't find any options for above syntax problem.

Thank you!

**Comment by prettydiff** [comment]
Two things:

* At this time there is no support for white space provided in attribute values unless the attribute is identified to be some kind of alternate template syntax apart from the vanilla XML/HTML code. The exact white space is preserved literally.  In this code example the indentation on line `  }"` makes the code wide enough to wrap the remaining bits onto a new line.
* There is no option at this time forcing every attribute onto a new line, but I could add such an option.

The first point above begs the question: *Why is the contents of {} in my Angular JS attribute not treated as JavaScript and beautified appropriately?*  To properly solve this problem I need two things:

* I have to know that the document is Angular JS code instead of just vanilla XML/HTML code.  It would be most convenient to be able to auto-magically infer this from the code somehow.  Otherwise I could create a new option allowing XML/HTML code to be described as an Angular document so that Angular friendly conventions are always applied, but that begs the question of when consuming applications should set this option for the beautifier library?

* Once I know with certainty, I don't to create regression issues for vanilla XML/HTML users, that the given code is Angular then I can supply Angular specific formatting considerations.  I need to know what the Angular specific code considerations are though.  A big giant list would be incredibly helpful.

Help me help you and I will provide you an Angular specific parser/beautifier for use in atom-beautify and elsewhere.

**Comment by Beingbook** [comment]
If I got you mean correctly, you need a AngularJS specific? Actually, I didn't see any standard document for AngularJS markup style. I just saw [JSX format](https://github.com/payscale/react-coding-standards#jsx) and it's just comfortable for me and not far from original HTML markup.

**Comment by Glavin001** [comment]
https://github.com/Glavin001/atom-beautify/pull/1520 has been merged.

Published to v0.29.18
